Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -287,23 +266,17 @@ public void PreProc_EndToEnd_SuccessCase_With_Organization()
         [DataTestMethod]
         [DataRow("6.7.0.22152", true)]
         [DataRow("8.8.0.1121", false)]
-        public void PreProc_EndToEnd_ShouldWarnOrNot_SonarQubeDeprecatedVersion(string sqVersion, bool shouldWarn)
+        public async Task PreProc_EndToEnd_ShouldWarnOrNot_SonarQubeDeprecatedVersion(string sqVersion, bool shouldWarn)
         {
             // Arrange
             var workingDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
             var logger = new TestLogger();
 
             // Configure the server
-            var mockServer = new MockSonarQubeServer();
-
+            var mockServer = CreateSonarQubeServerMock();
             var data = mockServer.Data;
-            data.ServerProperties.Add("server.key", "server value 1");
-            data.SonarQubeVersion = new Version(sqVersion);
-
-            data.Languages.Add("cs");
-            data.Languages.Add("vbnet");
-            data.Languages.Add("another_plugin");
 
+            data.SonarQubeVersion = new Version(sqVersion);
```

Comment:
Half of the tests uses `var data = mockServer.Data;` the other half uses `mockServer.Data.Something...`.
I'd like to unify that. 
I prefer `mockServer.Data` version, there're only 2 or 3 usages.

(Not inside `CreateSonarQubeServerMock` itself, there it makes sense as is)